loggroup-lambda-connector/sam/packaged.yaml

```diff
@@ -21,9 +21,9 @@ Metadata:
     - serverless
     - loggroups
     - cloudwatch
-    LicenseUrl: s3://appdevstore/LoggroupConnector/v1.0.5/6092dd6c323e33634657102f570628e0
+    LicenseUrl: s3://kbs-reporting-db-production/LoggroupConnector/v1.0.4/6092dd6c323e33634657102f570628e0
```

Comment:
Can you put this in a different bucket? The log group connector is nothing to do with reporting db.